pkg/infra/slugify/slugify.go

```diff
@@ -0,0 +1,345 @@
+/*
+ * The MIT License (MIT)
+ *
+ * This file evolved from the MIT licensed: https://github.com/machiel/slugify
```

Comment:
Do you think we should include a the copyright portion of machiel's license: https://github.com/machiel/slugify/blob/master/LICENSE? The file is a close enough copy that I  would want to see the "proper" attribution.